src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -282,23 +282,23 @@ def test_WINNF_FT_S_SIQ_4(self):
     """
     # Trigger SAS to load DPAs
     self._sas_admin.TriggerLoadDpas()
-    # Trigger SAS to de-active all the DPAs 
-    self._sas_admin.TriggerBulkDpaActivation({'activate':False})
+    # Trigger SAS to de-active all the DPAs
+    self._sas_admin.TriggerBulkDpaActivation({'activate': False})
     # Trigger SAS to active one DPA on channel c
     frequency_range = {
       'lowFrequency': 3620000000.0,
```

Comment:
Not really related to the changes introduced in this PR, just happened to find that the frequencies here are not integer values as we already agreed to use in all other test codes. Please help making the change if feasible, or otherwise we will need another round of cleanup. Thanks.